security-analysis/security-analysis-api/src/main/java/com/powsybl/security/results/BusResult.java

```diff
@@ -12,10 +12,11 @@
 import java.util.Objects;
 
 /**
+ * Contains state variables of a bus. Note that the busId depends on the topology used for the computation: it can be the
```

Comment:
Similarly `Contains state variables of a bus which is monitored during security analysis through a {@link StateMonitor}`?